### PR TITLE
Added : support for intel compiler

### PR DESCRIPTION
**Pull request by handrake0724**
intel compiler support is added.

### CODE REVIEW

**Comment by rizsotto** [comment]
thanks, but broken build, can't merge... please remove lines and do not comment out.

**Comment by rizsotto** [comment]
the build is still broken. please look at the `.travis.yml` file what needs to be passed, before your do send another one. and please also check the https://travis-ci.org/rizsotto/Bear/pull_requests for the build bot results. thanks!

**Comment by handrake0724** [comment]
I guess the problem is caused by bug94 test.
I don't know how to correct it. Maybe you can help me out.

File: bear/main.py.in [hunk 1]
```diff
@@ -178,7 +178,8 @@ def format_entry(entry):
     atoms = classify_parameters(entry['command'], abspath)
     if atoms['action'] <= Action.Compile:
         for source in atoms['files']:
-            compiler = 'c++' if atoms['c++'] else 'cc'
```
Comment (rizsotto) [inline bear/main.py.in]: this line shall stay!

**Comment by rizsotto** [comment]
the problem that you were try to rewrite some logic, instead of just extend the regexp lines.

**Pull request closed without merging**